### ISSUE TITLE
Reassurance module not translated

### ISSUE THREAD

**Issue by dreyser** [PrestaShop/PrestaShop#27503]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Translations are properly edited and successfully saved, however in the front office it still shows the original text in english, I even changed english version text to a different one and same results not showing changes

### Expected behavior

Translations properly showing

### Steps to reproduce

1.Install version 1.7.8.3
2.Configure customer reassurance module with translations
3.Go to any product page
4. Reassurance module text show up in english

### PrestaShop version(s) where the bug happened

1.7.8.3

### PHP version(s) where the bug happened

7.4.27

### If your bug is related to a module, specify its name and its version

Reassurance

**Comment by khouloudbelguith** [comment]
Hi @dreyser,

I did not manage to reproduce the issue with PS1783 using the blockreassurance v5.0.0.
![image](https://user-images.githubusercontent.com/16067358/151518336-835e6702-650b-403f-99a6-ef717d2622b9.png)

I attached a screen record
https://user-images.githubusercontent.com/16067358/151518083-ee3a43d2-61ea-411a-9308-3289a93b61aa.mp4


What is version of the module blockreassurance did you use?

Thanks!
